src/families/bitcoin/bridge/libcore.js

```diff
@@ -135,6 +135,7 @@ const prepareTransaction = async (
 };
 
 const getCapabilities = () => ({
+  canDelegate: false,
```

Comment:
Maybe it should be canStake

Reply:
ok let's keep this canDelegate for now. there is a clear semantic of delegation atm. we'll see later what we generalize.